kore/test/Test/Kore/Step/Simplification/Ceil.hs

```diff
@@ -375,27 +373,39 @@ test_ceilSimplification =
                 , domainValueChild = mkStringLiteral "a"
                 }
         assertEqual "ceil(1)" expected actual
-    , testCase "ceil with map domain value" $ do
-        -- maps assume that their keys are relatively functional, so
-        -- ceil({a->b, c->d}) = ceil(b) and ceil(d)
-        let
-            expected = OrPattern.fromPatterns
-                [ Conditional
-                    { term = mkTop_
-                    , predicate =
-                        makeAndPredicate
-                            (makeCeilPredicate fOfB)
-                            (makeCeilPredicate gOfB)
-                    , substitution = mempty
-                    }
-                ]
-        actual <- makeEvaluate
-            Conditional
-                { term = Mock.builtinMap [(fOfA, fOfB), (gOfA, gOfB)]
-                , predicate = makeTruePredicate
-                , substitution = mempty
-                }
-        assertEqual "ceil(map)" expected actual
+    , testGroup "Builtin.Map"
+        [ testCase "concrete partial keys" $ do
+            -- maps assume that their keys are relatively functional, so
+            -- ceil({a->b, c->d}) = ceil(b) and ceil(d)
```

Comment:
I guess that this is not directly related to this PR, but, since this instance of this comment showed up here, I'm going to mention this:

https://github.com/kframework/kore/issues/1077

I.e. this test is probably somewhat broken (as is the original that was used as a base for this), and using constructors or domain values as keys below may be better.